### PR TITLE
SAS-86 #create AT for initialize system uc

### PR DESCRIPTION
**Pull request by AmitNachimovitz**

### CODE REVIEW

File: src/test/java/com/SAS/acceptanceTests/systemControllerAT.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            userName = "admin";
+            System.out.println(userName);
+            System.out.println("Password:");
+            password = "admin123";
+            System.out.println("********");
+            System.out.println("Full Name:");
+            fullName = "Admin Admin";
+            System.out.println("fullName");
+            systemController.createSystemAdmin(userName, password, fullName);
+            System.out.println("The setup ended successfully");
```
Comment (yaarm) [inline src/test/java/com/SAS/acceptanceTests/systemControllerAT.java]: Maybe add asserts that both systems are up and the user exist.

File: src/test/java/com/SAS/acceptanceTests/systemControllerAT.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        //The user select the external systems he want to connect
+        System.out.println("Please select which external systems you want to connect to:");
+        System.out.println(systemController.showAvailableExternalSystem());
+        userChoose = "Acounting";
+        System.out.println(userChoose);
+        if (systemController.addExternalSystem(userChoose)) {
+            system1 = true;
+        }
+        else{
+            System.out.println("No External System with this name was found");
```
Comment (yaarm) [inline src/test/java/com/SAS/acceptanceTests/systemControllerAT.java]: Just add an assert that the system isn't up.

**Review by yaarm — CHANGES_REQUESTED** [review]
Looks great! 
Just add asserts.

File: src/test/java/com/SAS/acceptanceTests/systemControllerAT.java (hunk 1 above)
Comment (AmitNachimovitz) [reply src/test/java/com/SAS/acceptanceTests/systemControllerAT.java]: I added asserts

**Review by yaarm — APPROVED** [review]

**Pull request merged**